### DISCUSSION TITLE
how to make ".\target_name.exe" works after build using "xmake"?

### DISCUSSION THREAD

**Discussion by liwei46** [xmake-io/xmake#4833] [Q&A] [answered]
Hi,
I build a program in windows (with visual studio 2019) using xmake, and "xmake run target_name" works.  But ".\target_name.exe" not working.

Because I need to share the executable to other people that do not have xmake installed. 

Is it possible to make ".\target_name.exe" works? 
Thanks a lot.

**Comment by waruqi** [comment, marked as answer]
```console
xmake install -o output
./output/bin/target.exe
```

**Comment by liwei46** [comment]
> Hi, I build a program in windows (with visual studio 2019) using xmake, and "xmake run target_name" works. But ".\target_name.exe" not working.
> 
> Because I need to share the executable to other people that do not have xmake installed.
> 
> Is it possible to make ".\target_name.exe" works? Thanks a lot.

maybe also because my program cannot find some "dll" files, while "xmake run" can find them.
